src/Modules/SharedModules/components/DeleteData/DeleteData.tsx

```diff
@@ -0,0 +1,15 @@
+ import noData from '../../../../assets/images/no-data.png'
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  export default function DeleteData({deleteItem}) {
```

Comment:
The name `delteItem` feels like a function, but from the useage here indicate that it is a string variable containing the itemName. We can rename it to `itemName` for better readability and to avoid any confusion.